src/test/java/edu/ucla/library/prl/harvester/JobRequestsFT.java

```diff
@@ -475,8 +487,8 @@ void testUpdateBeforeAdd(final Vertx aVertx, final VertxTestContext aContext) {
         final Future<HttpResponse<Buffer>> updateJob;
 
         try {
-            job = TestUtils.getRandomJob(myInstitutionID);
-        } catch (final MalformedURLException | ParseException details) {
+            job = TestUtils.getJob(myInstitutionID, myTestProviderBaseURL, List.of(TestUtils.SET2));
+        } catch (final ParseException details) {
             aContext.failNow(details);
             return;
```

Comment:
Just curious, why use the return instead of putting the result of the code in the try?

Reply:
I figured because the rest of it is exception-free, it should be left outside. But I don't really have a strong opinion on this -- I think I've gone back and forth on whether to keep it all together or to separate.

Reply:
checked exception-free, ofc